src/components/LandingText/style.module.scss

```diff
@@ -0,0 +1,96 @@
+@use '../../styles/vars.scss' as vars;
+
+.container {
+  display: flex;
+  flex-direction: column;
+  gap: 1.813rem;
```

Comment:
No need to act on this but these numbers are oddly precise, are they from Figma or something?